plugins/pulp_docker/plugins/migrations/0003_tag_schema_change.py

```diff
@@ -0,0 +1,20 @@
+from pulp.server.db.connection import get_collection
+
+
+def migrate(*args, **kwargs):
+    """
+    Add schema_version to the tag collection.
+    """
+
+    schema_version_key = 'schema_version'
+    collection = get_collection('units_docker_tag')
+    # drop old index due to unit_keys fields change
+    index_info = collection.index_information()
+    old_index = 'name_1_repo_id_1'
```

Comment:
no i was wrong, the new index is created at this point, once all the migration are done
https://github.com/pulp/pulp/blob/master/server/pulp/server/db/manage.py#L147

So i think we are safe here.